staging/src/k8s.io/apimachinery/pkg/util/cache/expiring_test.go

```diff
@@ -101,6 +101,17 @@ func TestExpiration(t *testing.T) {
 	if _, ok := c.Get("a"); !ok {
 		t.Fatalf("we should have found a key")
 	}
+
+	// Check getting an expired key with and without AllowExpiredGet
+	c.Set("b", "b", time.Second)
+	fc.Step(2 * time.Second)
+	if _, ok := c.Get("b"); ok {
+		t.Fatalf("we should not have found b key")
+	}
+	c.AllowExpiredGet = true
```

Comment:
add a Set to be sure this properly clears the data even for the `AllowExpiredGet` case.

Reply:
Done.